scripts/funcotator/data_sources/gnomAD/createGnomadAlleleFreqTsv.wdl

```diff
@@ -0,0 +1,114 @@
+    String out_file_name
+
+    String gatk_docker
+
+    Int?  mem_gb
+    Int? preemptible_attempts
+    Int? disk_space_gb
+    Int? cpu
+    Int? boot_disk_size_gb
+
+    call CreateGnomadAlleleFreqTsvTask {
+        input:
+            gnomAD_file               = gnomAD_file,
+            out_file_name             = out_file_name,
+            gatk_docker               = gatk_docker,
+            mem_gb                    = mem_gb,
+            preemptible_attempts      = preemptible_attempts,
+            disk_space_gb             = disk_space_gb,
+            cpu                       = cpu,
+            boot_disk_size_gb         = boot_disk_size_gb
+    }
+
+    output {
+        File vcf_file = CreateGnomadAlleleFreqTsvTask.tsvOut
+    }
+}
+
+
+task CreateGnomadAlleleFreqTsvTask {
+
+    File gnomAD_file
+    String out_file_name
+
+     # ------------------------------------------------
+     # runtime
+     String gatk_docker
+     Int? mem_gb
+     Int? preemptible_attempts
+     Int? disk_space_gb
+     Int? cpu
+     Int? boot_disk_size_gb
+
+     # ------------------------------------------------
+     # Get machine settings:
+     Boolean use_ssd = false
+
+     # You may have to change the following two parameter values depending on the task requirements
+     Int default_ram_mb = 1024 * 3
+     # WARNING: In the workflow, you should calculate the disk space as an input to this task (disk_space_gb).  Please see [TODO: Link from Jose] for examples.
+     Int default_disk_space_gb = 100
+
+     Int default_boot_disk_size_gb = 15
+
+     # Mem is in units of GB but our command and memory runtime values are in MB
+     Int machine_mem = if defined(mem_gb) then mem_gb * 1024 else default_ram_mb
+     Int command_mem = machine_mem - 1024
+
+     # ------------------------------------------------
+     # Run our command:
+     command <<<
+         set -e
+
+        startTime=`date +%s.%N`
+        echo "StartTime: $startTime" > timingInformation.txt
+
+         cat ${gnomAD_file} | sed 's#^\([0-9X]*\)\t\([0-9]*\)\t\(.*\)\t\([ATGCN]*\)\t\([ATGCN,]*\)\t.*;AF=\([e0-9\.+\-]*\);.*#\1 \2 \3 \4 \5 \6#g' > ${out_file_name}
+
+         endTime=`date +%s.%N`
+         echo "EndTime: $endTime" >> timingInformation.txt
+         elapsedTime=`echo "scale=5;$endTime - $startTime" | bc`
+         echo "Elapsed Time: $elapsedTime" >> timingInformation.txt
+     >>>
+
+     # ------------------------------------------------
+     # Runtime settings:
+     runtime {
+         docker: gatk_docker
+         memory: machine_mem + " MB"
+         disks: "local-disk " + select_first([disk_space_gb, default_disk_space_gb]) + if use_ssd then " SSD" else " HDD"
+         bootDiskSizeGb: select_first([boot_disk_size_gb, default_boot_disk_size_gb])
+         preemptible: 0
+         cpu: select_first([cpu, 1])
+     }
+
+     # ------------------------------------------------
+     # Outputs:
+     output {
+         File tsvOut = "${out_file_name}"
```

Comment:
Can you use a more descriptive variable name?

Reply:
Sure.